mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
```

Comment:
Here is the review:

In the `Order.deleteOne` method, it's not handling the case where `req.params.orderId` is not a valid ObjectId. It should be validated before passing it to the `deleteOne` method to prevent any potential errors. Additionally, it's not checking if the `orderId` is present in the request params before attempting to delete the order. This could lead to errors if the `orderId` is not provided.